probdists/Gammadistribution.py

```diff
@@ -102,6 +102,34 @@ def plot_bar_pdf(self, points=25):
         plt.show()
         return x, y
 
+    def cdf_lower(self, x):
+        """
+        Cumulative density function calculator for the Gamma distribution.
+            Args:
+                x (float): point for calculating the cumulative distribution function
+            Returns:
+                float: lower cumulative distribution function output (1-cdf)
+        """
+        #initiate cdf variable
+        cdf = 0
+        for i in range (self.k):
+            cdf += (math.pow((x / self.theta), i) * math.exp(-1 * x / self.theta)) / math.factorial(i) 
+        return 1-cdf
+
```

Comment:
Well correct me if I'm wrong, but isn't this only applicable when k > 0? 
What do we do when k <= 0?